hms-app/src/components/CatDropdown.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+
+function CatDropdown() {
```

Comment:
We should avoid abbreviated names, right now this sounds like it is a dropdown for cats :cat:

Reply:
a cat dropdown sound great for me but i see the problem 😅